requirements.txt

```diff
@@ -1,5 +1,6 @@
 Flask-SQLAlchemy==2.3.2
 SQLAlchemy==1.2.14
 Flask-Script==2.0.6
-# py-ms==0.1.1
-git+https://github.com/python-microservices/pyms.git@master#egg=py-ms
\ No newline at end of file
+py-ms==0.1.1
+# git+https://github.com/python-microservices/pyms.git@master#egg=py-ms
+connexion[swagger-ui]==2.0.1
```

Comment:
Connexion viene dado por py-ms.
Por ahora usamos pyms de github hasta que generemos release

Reply:
Al final parece que no hay cambios que introducir en este proyecto, ¿verdad?
¿Se podría cancelar el PR o es que falta algo por hacer?

Reply:
Un segundo que quiero probar los decoradores que te dije antes y hacer un raise sin t-e.

Reply:
Si queréis cancelar este PR, no he podido avanzar y ya entramos en carga de trabajo por otro lado...
De todas maneras saldrá más adelante y ya nos pegamos con ello.

Reply:
Casi si, como seguro nos toca revisarlo de nuevo cuando demos con la clave ya abrimos de nuevo el PR.
De todos modos buen trabajo de investigación @pilamb ;)